src/Request/TableRequest.php

```diff
@@ -27,4 +27,36 @@ public static function createRecords(string $tableName, array $records): Request
             )
         );
     }
+
+    public static function readRecords(string $tableName, string $id): Request
+    {
+        Assert::that($tableName)
+            ->notEmpty();
+        Assert::that($id)
+            ->notEmpty();
+
+        return new self(
+            'GET',
+            $tableName . '/' . $id
+        );
+    }
+
+    public static function updateRecords(string $tableName, array $records, string $type): Request
+    {
+        Assert::thatAll($records)
+            ->keyExists('fields');
```

Comment:
this also need to assert the keyExists id on each record..
```suggestion
        Assert::thatAll($records)
            ->keyExists('fields')
            ->keyExists('id');
```